test/gtest/common/test_helpers.cc

```diff
@@ -364,6 +364,78 @@ void *mmap_fixed_address() {
     return (void*)0xff0000000;
 }
 
+sock_addr_storage::sock_addr_storage() : m_size(0), m_is_valid(false) {
+    memset(&m_storage, 0, sizeof(m_storage));
+}
+
+sock_addr_storage::sock_addr_storage(const ucs_sock_addr_t &ucs_sock_addr) :
+    m_size(ucs_sock_addr.addrlen), m_is_valid(false) {
+
+    if (sizeof(m_storage) < m_size) {
+        memset(&m_storage, 0, sizeof(m_storage));
+        return;
+    }
+    memcpy(&m_storage, ucs_sock_addr.addr, m_size);
+    m_is_valid = true;
+}
+
+void sock_addr_storage::set_sock_addr(const struct sockaddr &addr,
+                                      const size_t size) {
+    ASSERT_GE(sizeof(m_storage), size);
+    ASSERT_TRUE(ucs::is_inet_addr(&addr));
```

Comment:
we may want to just skip the test if there is no IP.

Reply:
Do you want to say that testing is passed because all our machines have valid IPoIB or ROcE address? I would expect don't initialize with wrong sockaddr

Reply:
if we dont, then a test is skipped. but now it will fail.

Reply:
`uct_test::set_interface_rscs` already does this check, so if this assertion fails that means there is a regression

Reply:
right.